kotlin-spark-api/3.2/pom_2.12.xml

```diff
@@ -36,20 +37,15 @@
       <artifactId>spark-streaming-kafka-0-10_${scala.compat.version}</artifactId>
       <version>${spark3.version}</version>
     </dependency>
-
-
-    <!-- Provided dependencies -->
     <dependency>
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-sql_${scala.compat.version}</artifactId>
-      <version>${spark3.version}</version>
-      <scope>provided</scope>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-sql_${scala.compat.version}</artifactId>
+        <version>${spark3.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-streaming_${scala.compat.version}</artifactId>
-      <version>${spark3.version}</version>
-      <scope>provided</scope>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-streaming_${scala.compat.version}</artifactId>
+        <version>${spark3.version}</version>
```

Comment:
Why it is not provided?

Reply:
Jupyter didn't work. If I removed all the provideds, it did work. Not sure why